testserver/server.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+import threading 
+import socket
+import select
+
+
+def consume_socket_content(sock, chunks=65536, timeout=0.5):
```

Comment:
There's no need to make `chunks` an argument here, we don't use it anywhere. Just set it in the method body.